karma.conf.js

```diff
@@ -17,12 +18,17 @@ module.exports = function(config) {
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
     coverageIstanbulReporter: {
-      dir: require('path').join(__dirname, 'coverage'), reports: ['html', 'lcovonly'],
+      dir: require('path').join(__dirname, 'coverage'),
+      reports: ['html', 'lcovonly'],
       fixWebpackSourcePaths: true
     },
+    angularCli: {
+      environment: 'production'
+    },
     reporters: ['progress', 'kjhtml'],
     port: 9876,
     colors: true,
+    logLevel: config.LOG_INFO,
```

Comment:
why you need usage logLevel instead?

```ts
browserConsoleLogOptions: {
  level: CI ? 'error' : 'debug'
}
```

Reply:
@splincode these two are with different scope. loglevel is for karma internal log, browserConsoleLogOptions is for reporter.

Reply:
but we will be moving to jest soon and karma internal log was not really needed